services/catalog/src/simcore_service_catalog/core/application.py

```diff
@@ -51,18 +53,26 @@ def init_app(settings: Optional[AppSettings] = None) -> FastAPI:
     )
     override_fastapi_openapi_method(app)
 
+    # STATE
     app.state.settings = settings
 
+    # PLUGIN SETUP
     setup_function_services(app)
 
-    # events
-    app.add_event_handler("startup", on_startup)
+    # EVENTS
+    async def _on_startup() -> None:
+        print(APP_STARTED_BANNER_MSG, flush=True)
+
+    async def _on_shutdown() -> None:
+        print(APP_FINISHED_BANNER_MSG, flush=True)
```

Comment:
I would like to delay that change because starletter (under fastapi) is deprecating start/shutdown events  and I would update them all at once https://fastapi.tiangolo.com/advanced/events/